src/helpers/lista-nominal/renderCell.tsx

```
@@ -0,0 +1,36 @@
+export type TagIconDetailsMap = Record<string, IconDetails>;
+
+export const renderDateTagCell = (
+  value: string | null,
+  iconDetailsMap: TagIconDetailsMap
+) => (
+  <>
+    {value ?? (
+      <TableTag
+        theme="pending"
+        text="Não realizada"
+        icon={iconDetailsMap["pending"]}
```

Biome (1.9.4)

[error] The computed expression can be simplified without the use of a string literal.

Unsafe fix: Use a literal key instead.


(lint/complexity/useLiteralKeys)